helm/idol-library-example-test/Chart.yaml

```diff
@@ -11,12 +11,12 @@
 
 apiVersion: v2
 name: idol-aci-test
-version: 0.1.0
+version: 0.2.0
 appVersion: "24.1"
```

Comment:
Updating this version doesn't matter too much as this isn't a published chart

Reply:
Should I change it back?